player_android.go

```diff
@@ -279,6 +290,19 @@ func (p *player) Write(data []uint8) (int, error) {
 }
 
 func (p *player) Close() error {
+	if p.audioTrack == nil {
+		return nil
+	}
+
 	runtime.SetFinalizer(p, nil)
-	return nil
+	err := jni.RunOnJVM(func(vm, env, ctx uintptr) error {
+		msg := C.releaseAudioTrack(C.uintptr_t(vm), C.uintptr_t(env), p.audioTrack)
```

Comment:
Updated. 5682ff6
Refer to the source codes around here, 
https://github.com/hajimehoshi/oto/blob/8ab1cc50f21ad05f93bf99dc8c44d6aa101c29c9/player_android.go#L209-L213
I inserted new line in condition on L.299-L300 too, do you prefer it?